proto/storage/alert.proto

```diff
@@ -128,7 +128,7 @@ message Alert {
 
     Enforcement               enforcement       = 6 [(gogoproto.moretags) = 'search:"Enforcement"'];
 
-    google.protobuf.Timestamp time              = 7 [(gogoproto.moretags) = "sensorhash:\"ignore\""  "search:\"Violation Time\""];
+    google.protobuf.Timestamp time              = 7 [(gogoproto.moretags) = "sensorhash:\"ignore\" search:\"Violation Time\""];
```

Comment:
better: `'sensorhash:"ignore" search:"Violation Time"'`

Reply:
Yes, there are both options in the file, I kept the previously used syntax for some reason.